README.md

```diff
@@ -168,7 +168,7 @@ tls:
 ```
```

Comment:
This needs additional command to create the agent-dynamic.conf file, otherwise the agent won't start up.  
```
sudo vi /etc/nginx-agent/agent-dynamic.conf
```

Reply:
@mtbChef added a commit to include the additional instructions.

Reply:
Ok got it, @craigell. Should we change it to `sudo touch /etc/nginx-agent/agent-dynamic.conf` since the next step after this is to edit the other conf file (`nginx-agent.conf`)?

Reply:
Sounds good.